### DISCUSSION TITLE
feat(deploy): Support for Preview Channels

### DISCUSSION THREAD

**Discussion by lacolaco** [angular/angularfire#2612] [Ideas]
Firebase Hosting finally supports the Preview Channel feature!

https://firebase.googleblog.com/2020/10/preview-channels-firebase-hosting.html

So I think `ng deploy` should allow developers to choose the channels. 

```
# Just a proposal inspired by https://github.com/marketplace/actions/deploy-to-firebase-hosting

{
  "builder": "@angular/fire:deploy",
  "options": {
    "channelId": "preview",
    "expires": "7d"
  } 
}
```

**Comment by jamesdaniels** [comment]
Absolutely aiming to support this. Need to bump the firebase-tools version in the next release.

**Comment by abdel-ships-it** [comment]
Any idea when this is ready? The documentation in [here](https://firebase.google.com/docs/hosting/github-integration) is somewhat out of sync with the out of the box setup of `ng add @angular/fire`

**Comment by Drofzz** [comment]
any update on this?
